proposals/templates/proposals/proposal_update_date_start.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block header_title %}
+    {% trans "Formulieren aanpassen" %} - {{ block.super }}
+{% endblock %}
+
+{% block content %}
+    <div class="uu-inner-container">
+        <div class="col-12">
+            <h2>
+                {% trans "Startdatum aanpassen" %}
+            </h2>
+            <p>
+                {% blocktrans trimmed with title=proposal.title ref_number=proposal.reference_number %}
+                Op deze pagina kan de startdatum worden aangepast van de aanvraag {{ title }}
+                (referentienummer <em>{{ ref_number }}</em>). <b>Let op!</b> Als de review al is afgerond,
+                wordt de nieuwe startdatum niet automatisch weergegeven in de PDF. Mocht u de PDF 
+                opnieuw willen genereren, neem hierover dan contact op met 
+                {% endblocktrans %}
+                <a href="mailto:portalsupport.gw@uu.nl">portalsupport.gw@uu.nl</a>.
+            </p>
+            <p>
+                {% trans "Vul de datum in als 'DD-MM-YYYY'." %}
+            </p>
```

Comment:
This is incorrect.

It might work, it will probably work when the locale is set to Dutch, but it might not always.

YYYY-MM-DD will always work

Or... Just use the custom year widget?

Reply:
I added the `DateField` widget from cdh core